docs/resources/dws_workload_plan_stage.md

```diff
@@ -0,0 +1,131 @@
+---
+subcategory: "Data Warehouse Service (DWS)"
+---
+
+# huaweicloud_dws_workload_plan_stage
+
+Manages a DWS workload plan stage resource within HuaweiCloud.
+
+## Example Usage
+
+```hcl
+variable "cluster_id" {}
+variable "plan_id" {}
+variable "stage_name" {}
+variable "pool_name" {}
+
+resource "huaweicloud_dws_workload_plan_stage" "test" {
+  cluster_id = var.cluster_id
+  plan_id    = var.cluster_id
+  name       = var.stage_nme
+  start_time = "01:00:00"
+  end_time   = "00:00:00"
+  queue_list {
+    name =  var.pool_name
+    configuration {
+      resource_name  = "cpu"
+      resource_value = 1
+    }
+    configuration {
+      resource_name  = "cpu_limit"
+      resource_value = 0
+    }
+    configuration {
+      resource_name  = "memory"
+      resource_value = 0
+    }
+    configuration {
+      resource_name  = "concurrency"
+      resource_value = 10
+    }
+    configuration {
+      resource_name  = "shortQueryConcurrencyNum"
+      resource_value = -1
+    }
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used.
+  Changing this creates a new resource.
+
+* `cluster_id` - (Required, String, ForceNew) Specifies the cluster ID of to which the workload plan belongs.
+  Changing this creates a new resource.
+
+* `plan_id` - (Required, String, ForceNew) Specifies the plan ID of to which the workload plan stage belongs.
+  Changing this creates a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the workload plan stage, which must be unique
+  and contains `3` to `28` characters, composed only of lowercase letters, numbers, or underscores (_),
+  and must start with a lowercase letter. Changing this creates a new resource.
+
+* `start_time` - (Required, String, ForceNew) Specifies the start time of the workload plan.
+  Changing this creates a new resource.
+
+* `end_time` - (Required, String, ForceNew) Specifies the end time of the workload plan.
+  Changing this creates a new resource.
+
+* `queue_list` - (Required, List, ForceNew) Specifies the workload queue configurations.
+  Changing this creates a new resource.
+  The [queue_list](#block--queue_list) structure is documented below.
+
+* `month` - (Optional, String, ForceNew) Specifies the execution months of the workload plan. Default to all months.
+  Changing this creates a new resource.
+
+* `day` - (Optional, String, ForceNew) Specifies the execution days of the workload plan. Default to all days.
+  Changing this creates a new resource.
```

Comment:
```
e.g. **...**
```
please provider the examples to teach users how to use them.

Reply:
done